Makefile

```diff
@@ -248,10 +248,10 @@ endif
 .PHONY: smoke-test
 smoke-test: ## run smoke tests
 	echo "Only run parts of the tests which we know are working. "
-	source .venv/bin/activate && (python3 -m pytest tests/test_integration.py || (cat /tmp/hopr-smoke-test_integration.log && false))
-	#source .venv/bin/activate && (python3 -m pytest tests/test_security.py || (cat /tmp/hopr-smoke-test_security.log && false))
-	#source .venv/bin/activate && (python3 -m pytest tests/test_websocket_api.py || (cat /tmp/hopr-smoke-test_websocket_api.log && false))
-	#source .venv/bin/activate && (python3 -m pytest tests/test_stress.py || (cat /tmp/hopr-smoke-test_stress.log && false))
+	source .venv/bin/activate && python3 -m pytest tests/test_integration.py
+	#source .venv/bin/activate && python3 -m pytest tests/test_security.py
+	#source .venv/bin/activate && python3 -m pytest tests/test_websocket_api.py
```

Comment:
Do you feel like uncommenting these and checking them as well then? They essentially need a single node setup I guess.

Reply:
Not for this PR. Afterwards, yes.